storage/s3.go

```diff
@@ -593,9 +584,28 @@ func (s *S3) MakeBucket(ctx context.Context, name string) error {
 	return err
 }
 
+// s3Session holds session.Session according to s3Opts
+// and it synchronizes access/modification.
+type s3Session struct {
+	sync.Mutex
+	sessions map[S3Options]*session.Session
+}
+
+// Sessions returns allSessions singleton.
+func Sessions() *s3Session {
```

Comment:
Do we need this function? `allSessions` is only accessed from the same package.

Reply:
I think we don't need, will remove it. I put there just in case we needed for e2e testing, but its not needed.